src/Controls/src/Core/Handlers/Shell/ShellHandler.Windows.cs

```diff
@@ -71,7 +71,10 @@ public static void MapCurrentItem(ShellHandler handler, Shell view)
 
 		public static void MapFlyoutBackground(ShellHandler handler, Shell view)
 		{
-			handler.NativeView.UpdatePaneBackground(view.FlyoutBackground ?? view.FlyoutBackgroundColor?.AsPaint());
+			handler.NativeView.UpdatePaneBackground(
+				!Brush.IsNullOrEmpty(view.FlyoutBackground) ?
```

Comment:
i think we repeat this validation in more places, we could create an extension method.